src/backend/distributed/citus.sql

```diff
@@ -309,11 +309,11 @@ CREATE FUNCTION worker_find_block_local_path(bigint, text[])
 COMMENT ON FUNCTION worker_find_block_local_path(bigint, text[])
     IS 'find an HDFS block''s local file path';
 
-CREATE FUNCTION worker_apply_shard_ddl_command(bigint, text)
+CREATE FUNCTION worker_apply_shard_ddl_command(bigint, text, text)
```

Comment:
We need to create an update script for this. 

Look at 
https://github.com/citusdata/citus/blob/master/src/backend/distributed/citus--5.1-2--5.1-3.sql

You can use update instead of drop and create.